index.js

```diff
@@ -269,10 +269,26 @@ var PgDriver = Base.extend({
         var constraint = [],
             cb;
 
+        if ((spec.unique || spec.primaryKey) && spec.autoIncrement) {
```

Comment:
No I'm not advocating for it, it seems pretty useless :) I was just pointing out that its not a limitation enforced by postgres